src/Microsoft.AspNet.Mvc.ModelBinding/ModelBindingContext.cs

```diff
@@ -56,36 +55,6 @@ public ModelBindingContext()
         public OperationBindingContext OperationBindingContext { get; set; }
 
         /// <summary>
-        /// Gets or sets the model associated with this context.
-        /// </summary>
-        /// <remarks>
-        /// The <see cref="ModelMetadata"/> property must be set to access this property.
-        /// </remarks>
-        public object Model
-        {
-            get
-            {
-                EnsureModelMetadata();
-                return ModelMetadata.Model;
-            }
-            set
-            {
-                IsModelSet = true;
-
-                EnsureModelMetadata();
```

Comment:
This one was a safety check that if the model metadata was not set, throw. In the new code we would get a Null ref in this case. 
Ill remove the Method.